### PR TITLE
NEW SHEET/ OFFICIAL / THE AETHYRBLOOD CHRONICLES

### PR DESCRIPTION
**Pull request by VJRolmasters**
# Submission Checklist
## Required

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

- The name of this game is: < THE AETHYRBLOOD CHRONICLES  > 
- The publisher of this game is: < HAPHAZARD PROJECTS  > 
- The name of this game system/family is: < DPS System ('dice pool scales') >

- [x] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

- [x] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.
- [x] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: < https://www.drivethrurpg.com/en/product/364098/The-Aethyrblood-Chronicles-Core-Rule-Book  >

# Changes / Description

Aethyrblood

### CODE REVIEW

File: The Aethyrblood Chronicles/sheet.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "html": "aethyrblood.html",
```
Comment (Anduh) [inline The Aethyrblood Chronicles/sheet.json:2]: I'm not sure if these things are case-sensitive, but your html & css files have capitalized names `Aethyrblood.html`, while in the `sheet.json` you wrote `aethyrblood.html`.

Comment (VJRolmasters) [reply The Aethyrblood Chronicles/sheet.json:2]: Thanks Anduh
I will change that, just in case.

**Comment by NorWhal** [comment]
Hey @VJRolmasters!
We can't open the preview image and github is saying it's an invalid source. Once that's fixed, we should be able to move forward!

**Comment by VJRolmasters** [comment]
> n't open the preview image and github is saying it's an invalid source. Once that's fixed, we should be able to move forward!

Hi,
Just deleted the image and upload the preview png file again, it seems to work.

**Pull request merged**